resources/web/style/openapi.pcss

```diff
@@ -0,0 +1,182 @@
+#guide {
```

Comment:
I think we should wrap all of this in a `<div class="openapi">` (which would require modifying the HTML template a bit), and nest everything in a `.openapi` block in the PCSS, to reduce the chances that these styles will conflict or override styles defined for the rest of the page.

Reply:
Nevermind, I see you literally just added this 10 seconds before I made my comment 🤣

Reply:
I woke up with that realization this morning, glad we're on the same wavelength!